crates/forge/tests/cli/test_optimizer.rs

```diff
@@ -1082,7 +1082,7 @@ contract CounterTest is Test {
 
     function test_Increment_In_Counter_With_Salt() public {
         CounterWithSalt counter = new CounterWithSalt{value: 111, salt: bytes32("preprocess_counter_with_salt")}(1);
-        assertEq(address(counter), 0x3Efe9ecFc73fB3baB7ECafBB40D3e134260Be6AB);
+        assertEq(address(counter), 0x223e63BE3BF01DD04f852d70f1bE217017055f49);
```

Comment:
These can be explained by differences in bytecode (like evm version) - we should make this test more robust